crates/argmin-math/Cargo.toml

```diff
@@ -15,11 +15,18 @@ exclude = []
 
 [dependencies]
 # nalgebra
+nalgebra_0_33 = { package = "nalgebra", version = "0.33", optional = true }
 nalgebra_0_32 = { package = "nalgebra", version = "0.32", optional = true }
 nalgebra_0_31 = { package = "nalgebra", version = "0.31", optional = true }
 nalgebra_0_30 = { package = "nalgebra", version = "0.30", optional = true }
 nalgebra_0_29 = { package = "nalgebra", version = "0.29", optional = true }
 
+# simba
+simba_0_6 = { package = "simba", version = "0.6", optional = true }
+simba_0_7 = { package = "simba", version = "0.7", optional = true }
+simba_0_8 = { package = "simba", version = "0.8", optional = true }
+simba_0_9 = { package = "simba", version = "0.9", optional = true }
```

Comment:
I'm not an nalgebra expert, could you explain to me why simba is needed as a dependency?

Reply:
simba is already a dependency of nalgebra, it's just that nalgebra 0.33 no longer re-exports certain traits used in this crate like `simba::scalar::ClosedAdd`.

Reply:
Actually taking a second look, I'm just replacing `ClosedAdd` with `ClosedAddAssign` which is reexported in 0.33, so I could remove simba and use the trait from nalgebra instead.

Reply:
If its not necessary I'd prefer it if it were removed, simply to avoid having to manage it.

Reply:
Ok, I've removed it.